edit-post/components/header/more-menu/test/index.js

```diff
@@ -8,6 +8,8 @@ import { mount } from 'enzyme';
  */
 import MoreMenu from '../index';
 
+jest.mock( '../../../../../packages/components/src/button' );
```

Comment:
That is a nigh-unusable import path? I know there's discussion around just using absolute paths everywhere (I brought it up in a chat and should really file an issue... so: #8190).

But this should be absolute if possible. It's really unusable to try to figure out what this is importing.

Reply:
Agreed, it took me some time to build them properly 😅 

In the first place, I want to get rid of those mocks completely with #7005 which will sort of resolve this issue :D